cmd/vela-hugo/hugo.go

```diff
@@ -91,18 +81,30 @@ func install(extendedBinary bool, customVer, defaultVer string) error {
 		return fmt.Errorf("not a valid version: %s", customVer)
 	}
 
-	// allow version usage with and without leading 'v'
-	customVer = strings.TrimPrefix(customVer, "v")
+	// get the version without leading "v",
+	// if it was supplied
+	verWithoutV := ver.String()
+
+	// check if the custom version requested
+	// is the default version
+	isDefaultVersion := strings.EqualFold(verWithoutV, defaultVer)
+
+	// are we using the included default
+	// (non-extended) version?
+	// if so, no need to download anything
+	if isDefaultVersion && !extendedBinary {
+		return nil
+	}
 
 	// let user know that a custom version
 	// was requested
-	if !isDefaultVersion && len(customVer) > 0 {
-		logrus.Infof("custom version requested (default is: %s): %s", defaultVer, customVer)
+	if !isDefaultVersion {
+		logrus.Infof("custom version requested (default is: %s): %s", defaultVer, ver.String())
```

Comment:
I noticed on line 86 above, we save the value of `ver.String()` with the variable `verWithoutV`.

Should we be re-using that variable here instead of calling `ver.String()` again?

Reply:
100%

Reply:
update pushed